src/Universum/Unsafe.hs

```diff
@@ -30,6 +30,10 @@ import Data.Maybe (fromJust)
 import Universum.Base (Int)
 import Universum.Function (flip)
 
+-- Non empty list definition for @liquidhaskell@.
+{-@ type NonEmptyList a = {xs:[a] | len xs > 0} @-}
```

Comment:
Okay, let's create some style guide convention for liquid haskell annotations and follow it. Like, spaces around `:` and spaces around `|`. Also, do you like spaces after `{` and before `}`?

Reply:
I like options with spaces.